wordpress-mobile/WordPress-Android
# Create tags and categories block

Fixes #8530 
This PR creates a new block with expandable tags and categories in the Stats. Clicking a tag redirects the use to the Tag detail in a `WebView`. Clicking "View more" opens a full list of tags and categories in the previous design. 

The expanded category should keep it's state on orientation changes.

To test:
* Open stats
* You see the Tags and Categories block
* Clicking on a category expands the tags in that category
* Clicking on a Tag opens a WebView
* Clicking on View More opens a new activity with a longer list of Tags and categories
